providers/zendesksupport/objectNames.go

```diff
@@ -1,21 +1,34 @@
 package zendesksupport
 
 import (
+	"github.com/amp-labs/connectors/common"
 	"github.com/amp-labs/connectors/common/handy"
 	"github.com/amp-labs/connectors/providers/zendesksupport/metadata"
 )
 
 // Supported object names can be found under schemas.json.
 var supportedObjectsByRead = metadata.Schemas.GetObjectNames() //nolint:gochecknoglobals
 
+// Grouping of ObjectName to response field name mappings by Module.
```

Comment:
Would be nice to comment/refer to some mock data that a viewer could look at to understand this better.